src/main/java/com/kruize/collection/CollectMetrics.java

```diff
@@ -353,37 +346,29 @@ private double getPreviousKruizeData(String recommendationURL)
             this.cpuQuery = cpuQuery;
         }
 
-        double getRss()
-        {
-            return rss;
-        }
-
-        double getCpu()
-        {
-            return cpu;
-        }
-
-        CurrentMetrics invoke() throws MalformedURLException
+        void invoke() throws MalformedURLException
         {
             try {
-                cpu = getValueForQuery(new URL(monitoringAgentEndPoint + cpuQuery));
+                double cpu = getValueForQuery(new URL(monitoringAgentEndPoint + cpuQuery));
                 System.out.println("CPU: " + cpu);
 
-                if (cpu < MIN_CPU)
-                    throw new ApplicationIdleStateException();
-
-                rss = getValueForQuery(new URL(monitoringAgentEndPoint + rssQuery));
+                double rss = getValueForQuery(new URL(monitoringAgentEndPoint + rssQuery));
                 System.out.println("RSS: " + rss);
 
                 //TODO Get network data from monitoring agent
                 double network = 0;
 
+                Kruize.applicationCpuUsed.labels(metrics.getNamespace(), metrics.getApplicationName())
+                        .set(cpu);
+                Kruize.applicationMemUsed.labels(metrics.getNamespace(), metrics.getApplicationName())
+                        .set(rss);
```

Comment:
Current metrics are now exported by CurrentMetrics.invoke(), instead of passing the instance to getKruizeRecommendations()